services/brig/schema/src/Main.hs

```diff
@@ -109,5 +109,7 @@ main = do
       V56.migration,
       V57.migration,
       V58.migration
+      -- FUTUREWORK: undo V41 (searchable flag); we stopped using it in
+      -- https://github.com/wireapp/wire-server/pull/964
```

Comment:
No users can now make themselves not-searchable, i.e., they will always end up on ES and be returned on the search API, right? The filtering of "who can see who" is simply part of the search query.

Reply:
This is also correct.